src/System/View/TemplatorFinder.php

```diff
@@ -0,0 +1,143 @@
+<?php
```

Comment:
better with

- `get` file get content.
- `getPaths` and `getEctension`

Reply:
I don't like add get, to get file content. lets Templator do this.